docs/user-guide/admin-cli.md

```diff
@@ -0,0 +1,141 @@
+[TOC]
+
+# Introduction
+In the context of the docker volume plugin for vSphere, each ESXi host manages multiple VMs, with
+each of them acting as a docker host. The docker engine on these hosts communicates with the docker
```

Comment:
Also, I think we need to be clear on terminology. Docker deals with volumes, but ESXi / Vmware "virtual disks", aka VMDKs. 
So when Docker requests a Volume creation, we create a virtual disk. The admin tool provide a view into virtual disks used as Docker  volumes.
I think using "volumes" in ESXi context without explicitly defining what we are talking about, will be very confusing. At the very least , if you do not feel "virtual disk" is any more clearer, 
we should use "Docker Volumes" everywhere.